src/sage/data_structures/pairing_heap.pxd

```diff
@@ -0,0 +1,82 @@
+# ******************************************************************************
+# Copyright (C) 2024 David Coudert <david.coudert@inria.fr>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#                  https://www.gnu.org/licenses/
+# ******************************************************************************
+
+# ==============================================================================
+# Interface to pairing heap data structure from ./pairing_heap.h
+# ==============================================================================
+
+from libcpp.pair cimport pair
+
+cdef extern from "./pairing_heap.h" namespace "pairing_heap":
+    cdef cppclass PairingHeap[TypeOfItem, TypeOfValue]:
+        PairingHeap() except +
+        PairingHeap(PairingHeap[TypeOfItem, TypeOfValue]) except +
+        bint empty()
+        void reset()
```

Comment:
This method is not defined in the .h file and not used anywhere

Reply:
right, I will remove it.